### PR TITLE
Joop session

### PR DESCRIPTION
**Pull request by JoopStark**
## Description
Created sessions service, facade, and controller
and create action

had to adjust schema it needs to be password_digest in the database

## Type of Change
- [ ] **fix**
- [x] **feat**
- [ ] **test**
- [ ] **refactor**
- [ ] **docs**

## Checklist

- [x] **Code has been self reviewed**
- [x] **Code runs without any errors**
- [ ] **Thorough testing has been implemented if adding feature**
- [ ] **All tests pass**

### CODE REVIEW

File: db/migrate/20231031175222_create_users.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ def change
       t.string :email
       t.integer :role
       t.string :movie_history, default: ""
-      t.string :password
+      t.string :password_digest
```
Comment (ttakahashi1591) [inline db/migrate/20231031175222_create_users.rb:8]: That makes sense!

**Review by ttakahashi1591 — COMMENTED** [review]
Thanks for getting the Sessions built out!!

File: app/models/user.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  has_secure_password
```
[the remark this answers is not in the document]
Comment (reedhillmar) [reply app/models/user.rb:2]: We're definitely going to have merge conflicts, but should be an easy one to fix

File: db/migrate/20231031175222_create_users.rb (hunk 1 above)
Comment (reedhillmar) [reply db/migrate/20231031175222_create_users.rb:8]: Again, this change was already made on our side, hopefully no merge conflict

**Review by reedhillmar — APPROVED** [review]
Well, have some merge conflicts because we repeated some work, but should be easy enough to resolve

**Pull request merged**